apps/website/foundation/icons/api.md

```diff
@@ -1,77 +1,4 @@
 ---
 title: API
+permalink: /storybook/core/?path=/story/components-icon-getting-started--page
```

Comment:
Little hack that can be used to redirect the user to storybook when the origin of the website is `clarity.design`